app/models/orchestration_template.rb

```diff
@@ -14,11 +14,14 @@ class OrchestrationTemplate < ApplicationRecord
 
   validates :md5,
             :uniqueness => {:scope => :draft, :message => "of content already exists (content must be unique)"},
-            :unless     => :draft?
+            :if         => :unique_md5?
   validates_presence_of :name
 
   before_destroy :check_not_in_use
 
+  attr_accessor :remote_proxy
```

Comment:
it activates the calls to tacker, so the Template model starts to act as a remote proxy. Not sure how to pack this call to a queue. But if the provider call doesn't succeed, the template does;t get created and user is notified